src/mumble/Overlay.cpp

```diff
@@ -213,6 +213,7 @@ Overlay::~Overlay() {
 
 void Overlay::setActive(bool act) {
 	if (!m_initialized.load()) {
+		createPipe();
```

Comment:
**question:** Before aac3214 the createPipe logic was executed after platformInit and forceSettings instead of before. Is there a reason for this change?

Personally I like to keep stuff as close to the previous logic as possible unless there is a reason for a change, and/or the consequences can be interpreted or verified. Because of possible unforeseen consequences.

From what I can see createPipe creates the qlsServer. forceSettings will call reset and update on any connected clients.  Effectively this change may now call reset and update on "new" clients (that may not have received or shown data before). I do not know if that makes a difference. Maybe it is just unnecessary clearing of state. Or maybe such a case is not handled cleanly on the other end.

Reply:
This is actually only my own sloppiness. I just put the call in there somewhere without too much thought.
I agree that it's a good idea to move it after `platformInit`